src/client/js/listing.js

```diff
@@ -16,25 +16,24 @@
  *  limitations under the License
  *
  */
- (function() {
-  let database = firebase.database();
-  firebase.database().ref('/products/').once('value').then(function(snapshot) {
+export default function initializeProducts() {
+  firebase.database().ref('/products/').once('value').then((snapshot) => {
     populatePiesList(snapshot.val());
   });
 
   function populatePiesList(pies) {
-    let pieList = document.querySelector('#pies');
+    const pieList = document.querySelector('#pies');
```

Comment:
Note: All these `let` -> `const` changes are to satisfy our linter. I'll make it a mandatory part of the build process in a future PR, once we're happy with the state of our infra.

Reply:
Just please don't require lint on every file save, it's killing me;) Maybe a presubmit requirement rather than check on save?

Reply:
No worries, sounds good! Presubmit it is!

But you mention you're seeing this on every save? That's odd, because I currently have it as a completely separate task. Perhaps it's something to do with your editor config?

Reply:
It was set previously in gulp (there was no watch, so needed to restart server on save, which triggered linting).

Reply:
Ah, got it. Won't do that again, then :)